wp-includes/rest-api/endpoints/class-wp-rest-customize-partials-controller.php

```diff
@@ -0,0 +1,327 @@
+	 *
+	 * @since ?.?.?
+	 * @access public
+	 *
+	 * @see register_rest_route()
+	 */
+	public function register_routes() {
+
+		register_rest_route( $this->namespace, '/' . $this->rest_base, array(
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_items' ),
+				'permission_callback' => array( $this, 'get_items_permissions_check' ),
+				'args'                => $this->get_collection_params(),
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/(?P<partial>[\w-]+)', array(
+			'args' => array(
+				'partial' => array(
+					'description' => __( 'An alphanumeric identifier for the partial.' ),
+					'type'        => 'string',
+				),
+			),
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_item' ),
+				'permission_callback' => array( $this, 'get_item_permissions_check' ),
+				'args'                => array(
+					'context' => $this->get_context_param( array(
+						'default' => 'view',
+					) ),
+				),
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+	}
+
+	/**
+	 * Retrieves the partial's schema, conforming to JSON Schema.
+	 *
+	 * @since 4.?.?
+	 * @access public
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/schema#',
+			'title'      => 'partial',
+			'type'       => 'object',
+			'properties' => array(
+				'fallback_refresh' => array(
+					'description' => __( 'Whether to refresh the entire preview in case a partial cannot be refreshed.' ),
+					'type'        => 'boolean',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'id'              => array(
+					'description' => __( 'Identifier for the partial.' ),
+					'type'        => 'string',
+					'context'     => array( 'embed', 'view' ),
+					'readonly'    => true,
+				),
+				'container_inclusive' => array(
+					'description' => __( 'Whether the container element is included in the partial' ),
+					'type'        => 'boolean',
+					'context'     => array( 'embed', 'view' ),
+					'readonly'    => true,
+				),
+				'selector'        => array(
+					'description' => __( 'The jQuery selector to find the container element for the partial.' ),
+					'type'        => 'string',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'settings'        => array(
+					'description' => __( 'IDs for settings tied to the partial.' ),
+					'type'        => 'array',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'type'            => array(
+					'description' => __( 'Type of the partial.' ),
+					'type'        => 'string',
+					'context'     => array( 'view' ),
```

Comment:
Probably it would be a good idea to review all the `context` params in other endpoints as well, not sure how correct these are. Will create a separate PR for those.